Create backlog_cleanup.md (#386)
DXHeroes/knowledge-base-content: practices/backlog_cleanup.md

short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
tags:
  - backlog cleaning
  - agile
  - scrum
keywords:
  - backlog
  - cleaning
  - grooming
  - agile
  - scrum
author_github_username: cadamini
---

# Backlog Cleanup

**TL;DR**

A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.

## What Is Backlog Cleanup  

Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.

![Product backlog](/files/backlog_cleanup.webp)
[Name](https://img.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_23-2148513829.jpg)

## Why You Might Want Backlog Cleanup

Both the prioritiation and the size of a product backlog is key. Your team will most probably work on the top items in the list. 

> When product backlogs grow, it will be harder to see the relation between items. Backlog cleanup will help you to keep the overview.

Author: ```suggestion
When backlogs grow, it will be harder to see the relationship between items. Backlog cleanup will help you to keep track.
```

## Problems Backlog Cleanup Helps to Solve

- Getting a better overview of important tasks
- Not loosing time due to a long and confusing backlog

## How to Implement Backlog Cleanup

- **Go through the product backlog on a regular basis:** For example, once a week or month: This process is called Grooming. The frequency depends on your the number of new cards and how often strategy changes happen.
- **Find a good upper limit:** Find a good number of backlog items that you can deal with. You can store your ideas separately and review them after your next strategy meeting. 
- **Group related items:** To find relations between items, you can try to group them in Epics. Epics are your big topics that contain smaller pieces that are required to reach a goal or ship a bigger feature.

> - **Prioritize your backlog and make the top items ready to work on:** In each sprint, your team will work on the items at the top of your backlog. They will need to know what to do.

Author: ```suggestion
- **Prioritize your backlog:** The higher the items in the list, the more important they are.
- **Make enough items ready to work on:** In each sprint, your team works on the items at the top of your backlog. The team needs to know what to do.
```

## Common Pitfalls

> - You will see items that are not ready to work on. Remove these items and take them to your next Grooming meeting (again).

Author: ```suggestion
- You will often see items that are not ready to work on yet. Remove these items and take them (again) to your next grooming meeting.
```

> - There may be ideas that are not in line with the current product strategy. Lower their priority or remove them from your backlog. 

Author: ```suggestion
- There may be ideas that do not fit the current product strategy. Don't forget to lower their priority or remove them from your backlog.
```

---
slug: backlog-cleaning
stages:
  - development
short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
tags:
  - backlog cleaning
  - agile
  - scrum
keywords:
  - backlog
  - cleaning
  - grooming
  - agile
  - scrum
author_github_username: cadamini
---

# Backlog Cleanup

**TL;DR**

A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.

## What Is Backlog Cleanup  

> Backlog Cleanup describes the process of managing issues in your product backlog. Your backlog may be full of old feature ideas.
> After some strategy meetings in the last months/quarters, your feature ideas may be out of scope, since the product strategy has shifted in a different direction.

Author: ```suggestion
Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.
```

## Problems Backlog Cleanup Helps to Solve

- Getting a better overview of important tasks
- Not loosing time due to a long and confusing backlog

## How to Implement Backlog Cleanup

- **Go through the product backlog on a regular basis:** For example, once a week or month: This process is called Grooming. The frequency depends on your the number of new cards and how often strategy changes happen.
- **Find a good upper limit:** Find a good number of backlog items that you can deal with. You can store your ideas separately and review them after your next strategy meeting. 
- **Group related items:** To find relations between items, you can try to group them in Epics. Epics are your big topics that contain smaller pieces that are required to reach a goal or ship a bigger feature.
- **Prioritize your backlog:** The higher the items in the list, the more important they are.
- **Make enough items ready to work on:** In each sprint, your team works on the items at the top of your backlog. The team needs to know what to do.

## Common Pitfalls

- You will often see items that are not ready to work on yet. Remove these items and take them (again) to your next grooming meeting.

> - There may be ideas that do not fit the current product strategy. Don't forget to lower their priority or remove them from your backlog.

Author: ```suggestion
- There may be ideas that do not fit the current product strategy. Don't forget to lower their priority or remove them from your backlog.
- When you have too many backlog items to work on, it becomes difficult to work with your backlog.
```

---
slug: backlog-cleaning
stages:
  - development
short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
tags:
  - backlog cleaning
  - agile
  - scrum
keywords:
  - backlog
  - cleaning
  - grooming
  - agile
  - scrum
author_github_username: cadamini
---

# Backlog Cleanup

**TL;DR**

> To not loose the overview, you must regularly remove obsolete backlog items that have been created during the overall lifecycle of a project. 
> A well-managed backlog results in a better overview of your important open issues and features, faster plannings, and a cleaner scope.

Author: ```suggestion
A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
```

## What Is Backlog Cleanup  

Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.

![Product backlog](/files/backlog_cleanup.webp)

> [Name](https://img.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_23-2148513829.jpg)

Reviewer: @cadamini Line 29 is the image itself, and line 30 should be a source of the image if licensed under some license that requires mentioning the author. (e.g. CC v4).

Please remove the line 30 or just set an URL to the article/page with the source image.

Author: ```suggestion
![Product backlog](/files/backlog_cleanup.webp)
```

See above. Completely free image. Just remove this if this is ok.

## Why You Might Want Backlog Cleanup

Both the prioritiation and the size of a product backlog is key. Your team will most probably work on the top items in the list. 
When backlogs grow, it will be harder to see the relationship between items. Backlog cleanup will help you to keep track.

## Problems Backlog Cleanup Helps to Solve

- Getting a better overview of important tasks
- Not loosing time due to a long and confusing backlog

## How to Implement Backlog Cleanup

- **Go through the product backlog on a regular basis:** For example, once a week or month: This process is called Grooming. The frequency depends on your the number of new cards and how often strategy changes happen.

> - **Find a good upper limit:** Find a good number of backlog items that you can deal with. You can store your ideas separately and review them after your next strategy meeting. 

Author: ```suggestion
- **Define an upper limit for the number of items:** You can store your ideas separately and review them after your next strategy meeting. 
```

Not sure about this point. Now duplicated in pain points and rewritten here. What do you think? Worth the repetition?

Reviewer: I don't think repetition is a problem. :) I'll leave it up to you.

> cleaning your product backlog
> 
> ---

Author: ```suggestion
---
```

slug: backlog-cleaning
stages:
  - development
short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
tags:
  - backlog cleaning
  - agile
  - scrum
keywords:
  - backlog
  - cleaning
  - grooming
  - agile

>   - scum

Author: ```suggestion
  - scrum
```

author_github_username: cadamini
---

# Backlog Cleanup

**TL;DR**

To not loose the overview, you must regularly remove obsolete backlog items that have been created during the overall lifecycle of a project. 
A well-managed backlog results in a better overview of your important open issues and features, faster plannings, and a cleaner scope.

## What Is Backlog Cleanup  

Backlog Cleanup describes the process of managing issues in your product backlog. Your backlog may be full of old feature ideas.
After some strategy meetings in the last months/quarters, your feature ideas may be out of scope, since the product strategy has shifted in a different direction.

> ![Product backlog](/files/your image)

Author: ```suggestion
![Product backlog](/files/backlog_cleanup.webp)
```
